pkg/config/config.go

```diff
@@ -1024,6 +1024,8 @@ func InitConfig(config Config) {
 	// Serverless Agent
 	config.BindEnvAndSetDefault("serverless.logs_enabled", true)
 	config.BindEnvAndSetDefault("enhanced_metrics", true)
+	config.BindEnvAndSetDefault("capture_lambda_payload", false)
+	config.BindEnv("dotnet_tracer_home")
```

Comment:
Registered these two env vars to the config to suppress two of the warnings given by the extension (see below):
<img width="607" alt="image" src="https://user-images.githubusercontent.com/97111143/160867781-54a82f5d-82ea-4607-8665-4081350fc879.png">

Is there a reason these variables weren't registered before? `DD_DOTNET_TRACER_HOME` and `DD_TRACE_ENABLED` still aren't registered

Reply:
good question, it was introduced quite recently : https://github.com/DataDog/datadog-agent/pull/9646
and maybe we were not seeing that because of : https://github.com/DataDog/datadog-agent/pull/11269

Reply:
I've registered these four variables to suppress these warnings